packages/components/dropdown/src/Dropdown.doc.mdx

```diff
@@ -124,16 +124,22 @@ You can style this element directly, or you can use it as a wrapper to put an ic
 
 <Canvas of={stories.ItemIndicator} />
 
-### Trigger leading icon
+### States
```

Comment:
status

states are defined like: hover, focus, disabled, etc

Reply:
You are right, my brain always confuses the two